File: Aula02/teste.cpp

#include <iostream>
#include <string>

int main() {
    std::string nome;
    int idd;

    std::cout << "Informe seu nome: ";
    std::getline(std::cin, nome); //
    //getline: com ele eu consigo ler uma linha inteira incluindo espaços
    
    std::cout << "Informe sua idade: ";
    std::cin >> idd;

    // xibe a mensagem
    std::cout << "Olá " << nome << ", você tem " << idd << " anos." << std::endl;
    // endl = quebra de linha
    return 0;
}

File: Aula02/Teste.java
import java.util.Scanner;

public class Teste {
    public static void main(String[] args) {
        // Criação de um objeto Scanner para ler entrada do usuário
        Scanner scanner = new Scanner(System.in);

        System.out.print("Informe seu nome: ");
        String nome = scanner.nextLine();  // Lê uma linha (incluindo espaços)

      
        System.out.print("Informe sua idade: ");
        int idade = scanner.nextInt(); 

        System.out.println("Olá " + nome + ", Você tem " + idade + " anos.");

        // Fechar o scanner 
        scanner.close();
    }
}
